reactor-core/src/main/java/reactor/core/scheduler/BoundedElasticScheduler.java

```diff
@@ -142,6 +145,36 @@ public boolean isDisposed() {
 		return state.currentResource == SHUTDOWN;
 	}
 
+	@Override
+	public void init() {
+		SchedulerState<BoundedServices> b =
+				SchedulerState.init(new BoundedServices(this));
+		if (STATE.compareAndSet(this, INIT, b)) {
+			try {
+				b.currentResource.evictor.scheduleAtFixedRate(
+						b.currentResource::eviction,
+						ttlMillis, ttlMillis, TimeUnit.MILLISECONDS
+				);
+				return;
+			} catch (RejectedExecutionException ree) {
+				// The executor was most likely shut down in parallel.
```

Comment:
there have to be new IllegalStateException("Initializing a disposed scheduler is not permitted")

Reply:
Right, but actually the message should be different. I'm going with `throw new IllegalStateException("Scheduler disposed during initialization")`